server_installation/topics/profiles.adoc

```diff
@@ -60,6 +60,19 @@ endif::[]
 |Upload scripts through the {project_name} REST API
 |No
 |Deprecated
+
+|web_authn
+|W3C Web Authentication (WebAuthn)
+
+ifeval::[{project_community}==true]
+|Yes
+|Experimental
```

Comment:
@mposolda I don't know, if the _Supported level_ can be `Experimental` for project?!

Reply:
@mabartos I think it is fine and I don't have any better idea. So i would keep it like that :)

Reply:
@mabartos I think it is fine and I don't have any better idea. So i would keep it like that :)